app/code/Magento/Sales/Model/Order/Payment/Transaction.php

```diff
@@ -942,60 +942,55 @@ public function getCreatedAt()
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     public function setCreatedAt($createdAt)
     {
         return $this->setData(TransactionInterface::CREATED_AT, $createdAt);
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     public function setParentId($id)
     {
         return $this->setData(TransactionInterface::PARENT_ID, $id);
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     public function setOrderId($id)
     {
         return $this->setData(TransactionInterface::ORDER_ID, $id);
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     public function setPaymentId($id)
     {
         return $this->setData(TransactionInterface::PAYMENT_ID, $id);
     }
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     public function setIsClosed($isClosed)
     {
         return $this->setData(TransactionInterface::IS_CLOSED, $isClosed);
     }
 
     /**
-     * {@inheritdoc}
-     *
-     * @return \Magento\Sales\Api\Data\TransactionExtensionInterface|null
```

Comment:
Please, don't remove type hints for the method because the parent method has annotation with generic interface